python/packages/isce3/signal/rfi_process_evd.py

```diff
@@ -48,9 +48,9 @@ def run_slow_time_evd(
         time. Hence the standard (STD) deviation of multiple dominant EVs across slow time 
         defined by this parameter are compared. The one with the maximum STD is used for RFI
         Eigenvalue first difference computation.
-    num_rng_blks: int, default=1
-        Number of range bin blocks to be processed for EVD, default=1
-        When num_rng_blks=1, all range samples are used to compute EVD
+    num_samples_rng_blk: int
+        Number of range samples per range block for computation of sample covariance matrix.
+        default=256, in general num_samples_rng_blk > 2 x cpi_len is needed
```

Comment:
For the sake of consistency, let's put the default value on the same line as the parameter type, since that's what we've been doing elsewhere in this docstring.

```suggestion
    num_samples_rng_blk: int, default=256
        Number of range samples per range block for computation of sample covariance matrix.
        In general, num_samples_rng_blk > 2 x cpi_len is needed
```